File: firebase/functions/src/firestore/seat/Seat.ts
import {SeatStateType} from "./SeatStateType";

export interface Seat {
    name: string;
    minor: string;
    state: SeatStateType;
    isAvailable: boolean;
    userId: string | null;
    reserveEndTime: number | null;
    occupyEndTime: number | null;
}

File: core/model/src/main/java/io/github/jeddchoi/model/Seat.kt
package io.github.jeddchoi.model

data class Seat (
    val id : String = "",
    val name: String = "",
    val minor: String = "",
    val state: SeatStateType = SeatStateType.Empty,
    val isAvailable: Boolean = false,
    val userId: String? = null,
    val reserveEndTime: Long? = null,
    val occupyEndTime: Long? = null,
)

